app/kumactl/cmd/install/testdata/install-control-plane.dump-values.yaml

```diff
@@ -138,6 +138,37 @@ controlPlane:
     path: /
     # -- Each path in an Ingress is required to have a corresponding path type. (ImplementationSpecific/Exact/Prefix)
     pathType: ImplementationSpecific
+  
+  diagnostics:
+    service:
+      # -- Port on which Diagnostics Service is exposed on Node for service of type NodePort
+      nodePort: 30680
+
+  apiServer:
```

Comment:
There's already `controlPlane.service`, can we merge them together?

Reply:
How do you distinct http apiServer nodePort and https apiServer nodePort ?
Just give me an example, I'll follow.

Reply:
Yes, it's done